Gordon360/Models/ViewModels/MissingItemReportViewModel.cs

```diff
@@ -8,56 +9,59 @@ public class MissingItemReportViewModel
     {
         public int? recordID { get; set; }
 
-        public string firstName { get; set; }
+        public string? firstName { get; set; }
 
-        public string lastName { get; set; }
+        public string? lastName { get; set; }
 
         public string category { get; set; }
 
-        public string[]? colors { get; set; }
+        public string[] colors { get; set; }
 
         public string? brand { get; set; }
 
         public string description { get; set; }
 
         public string locationLost { get; set; }
 
-        public bool? stolen { get; set; }
+        public bool stolen { get; set; }
 
         public string? stolenDescription { get; set; }
 
-        public DateTime? dateLost { get; set; }
+        public DateTime dateLost { get; set; }
 
-        public DateTime? dateCreated { get; set; }
+        public DateTime dateCreated { get; set; }
 
-        public string phoneNumber { get; set; }
+        public string? phone { get; set; }
 
-        public string? altPhone { get; set; }
-
-        public string emailAddr { get; set; }
+        public string? email { get; set; }
 
         public string status { get; set; }
 
-        public string? adminUsername { get; set; }
+        public string submitterUsername { get; set; }
+
+        public string? submitterID { get; set; }
```

Comment:
Optional in the case of it being a guest?

Reply:
It's optional because we don't pass the ID to the frontend, but the ID is what is stored in the db.  So we remove the ID from the object before passing it to the frontend, and we fill in the ID before passing it to the db.